.github/workflows/Pyinstaller-windows.yml

```diff
@@ -0,0 +1,52 @@
+name: Package Application with Pyinstaller on Mingw64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: msys2/setup-msys2@v2
+      with:
+        msystem: MINGW64
+        install: >-
+          git
+          base-devel
+          zip
+        pacboy:
+          openssl
+          python-pip
+
+    - shell: msys2 {0}
+      run: |
+        git submodule update --init
+        python --version
+        python -m pip install pyinstaller
+        python -m pip install .
+        pacman -Ql | grep openssl | grep dll
```

Comment:
Could we include this in the Makefile ? Even though we cannot run it on a Linux machine

Reply:
I could, but this is Msys2 specific, which even on windows isn't that use. so I'm not too sure about putting this part outside the yaml. 
Especially as, one the debug remove, it's like a 3-line diff.